schema/check_generated_data.py

```diff
@@ -12,6 +12,10 @@
 import os.path
 import sys
 
+# To get commanlin arguments
+sys.path.append('../testdriver')
+from ddtargs import schemaArgs
```

Comment:
This looks suspicious

Reply:
I'll fix the comment. Note that this the same code used to import command arguments in other parts, e.g., verifier/verifier.py.

Reply:
Removed this code s not used.